test/scales/modifiedLogScaleTests.ts

```diff
@@ -91,7 +91,7 @@ describe("Scales", () => {
       scale.domainMin(minInMiddle);
       var requestedDomain2 = [-10, 10];
       scale.addExtentsProvider((scale: Plottable.Scales.ModifiedLog) => [requestedDomain2]);
-      scale._autoDomainIfAutomaticMode();
+      (<any>scale)._autoDomainIfAutomaticMode();
```

Comment:
Unless I'm mistaken `scale.addExtentsProvider` actually autoDomains by default, I don't think this line is necessary